src/any_agent/frameworks/any_agent.py

```diff
@@ -134,27 +134,18 @@ async def _load_tools(
 
     def _setup_tracing(self) -> None:
```

Comment:
We now follow the original design where setup_tracing only gets called in the AnyAgent __init__. We will still get those OTEL warnings when multiple AnyAgents are created in a single process, but this at least improves the design